micro_mg3_0.F90

```diff
@@ -287,10 +287,17 @@ module micro_mg3_0
 real(r8)           :: micro_mg_effi_factor
 real(r8)           :: micro_mg_iaccr_factor
 real(r8)           :: micro_mg_max_nicons
-
+      
 logical  :: remove_supersat ! If true, remove supersaturation after sedimentation loop
 logical  :: do_sb_physics ! do SB 2001 autoconversion or accretion physics
 
+real(r8)           :: vfactor = 1.0
+real(r8)           :: vfac_drop = 1.0   ! h1g, 2020-06-18
+real(r8)           :: vfac_ice  = 1.0   ! h1g, 2020-06-18
```

Comment:
there three module variables may need to be added to the `acc declare create` directive below, too.

Reply:
Added as a parameter